l10n_de_toponyms/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 IT IS AG <oca@itis.de>
+# Copyright 2018 Florian Kantelberg <florian.kantelberg@initos.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "German Toponyms",
+    "version": "11.0.1.0.0",
+    "author": '''IT IS AG Germany,
```

Comment:
Please don't use triple quote, as this adds unneeded spaces between each name

Reply:
@pedrobaeza damned we will correct that in a minute ;-) @fkantelberg you could you please give quick hands here